### ISSUE TITLE
Exceptions parsing PE SecurityCertificate

### ISSUE THREAD

**Issue by ryanmkurtz** [NationalSecurityAgency/ghidra#2363]
I tried to import a Windows 10 32-bit kernel32.dll and a 64-bit shell32.dll and both of them failed to load with this exception:

```
ERROR Error Importing File: Error importing file: kernel32.dll java.io.EOFException: EOF: unable to read 29992 bytes at 2914624
	at ghidra.app.util.bin.RandomAccessByteProvider.readBytes(RandomAccessByteProvider.java:145)
	at ghidra.app.util.bin.BinaryReader.readByteArray(BinaryReader.java:712)
	at ghidra.app.util.bin.format.pe.SecurityCertificate.initSecurityCertificate(SecurityCertificate.java:102)
	at ghidra.app.util.bin.format.pe.SecurityCertificate.createSecurityCertificate(SecurityCertificate.java:85)
	at ghidra.app.util.bin.format.pe.SecurityDataDirectory.parse(SecurityDataDirectory.java:119)
	at ghidra.app.util.bin.format.pe.DataDirectory.processDataDirectory(DataDirectory.java:80)
	at ghidra.app.util.bin.format.pe.SecurityDataDirectory.initSecurityDataDirectory(SecurityDataDirectory.java:57)
	at ghidra.app.util.bin.format.pe.SecurityDataDirectory.createSecurityDataDirectory(SecurityDataDirectory.java:47)
	at ghidra.app.util.bin.format.pe.OptionalHeaderImpl.processDataDirectories(OptionalHeaderImpl.java:348)
	at ghidra.app.util.opinion.PeLoader.load(PeLoader.java:124)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.doLoad(AbstractLibrarySupportLoader.java:347)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.importLibrary(AbstractLibrarySupportLoader.java:685)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.importLibrary(AbstractLibrarySupportLoader.java:635)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.loadLibraries(AbstractLibrarySupportLoader.java:259)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.loadProgram(AbstractLibrarySupportLoader.java:97)
	at ghidra.app.util.opinion.AbstractProgramLoader.load(AbstractProgramLoader.java:112)
	at ghidra.plugin.importer.ImporterUtilities.importSingleFile(ImporterUtilities.java:400)
	at ghidra.plugin.importer.ImporterDialog.lambda$7(ImporterDialog.java:349)
	at ghidra.util.task.TaskLauncher$1.run(TaskLauncher.java:90)
	at ghidra.util.task.Task.monitoredRun(Task.java:124)
	at ghidra.util.task.TaskRunner.lambda$1(TaskRunner.java:94)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by ryanmkurtz** [comment]
I think Ghidra is parsing multiple certificates wrong: 

https://github.com/NationalSecurityAgency/ghidra/blob/cd54d115cdaf3735badd81ec497828433c618692/Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pe/SecurityDataDirectory.java#L101-L132

This is how you are supposed to do it:

https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#the-attribute-certificate-table-image-only

**Comment by dev747368** [comment]
Can't dup.

**Comment by ryanmkurtz** [comment]
Did you do the 32-bit one in c:\windows\syswow64?

**Comment by dev747368** [comment]
It had 32 in the name.... :(
I'll try that one.

**Comment by ryanmkurtz** [comment]
Maybe it's because my windows 10 is very up to date. I'll get you the sample tomorrow.

**Comment by dev747368** [comment]
Don't worry... I was joshing you about the name of the shell32.dll.  I did get the wow64 version and was able to duplicate and fix the problem.

**Comment by dev747368** [comment]
should be fixed in f04c7fb2222a6440014bf6bd28d0dad70751664b
